src/component/Footer/style.css

```diff
@@ -1,185 +1,262 @@
 }
 
 .footer-container-mobile {
-    background-color: #187dff;
-    text-align: left;
-    padding-left: 15vw;
-    padding-top: 5vh;
-    padding-right: 15vw;
-    padding-bottom: 5vh;
-    width: 100vw;
-    margin: 0;
-    height: auto;
-}
-
-.left-bar-item {
-    display: flex;
-    flex-direction: column;
+  background-color: #187dff;
+  text-align: left;
+  padding-left: 15vw;
+  padding-top: 5vh;
+  padding-right: 15vw;
+  padding-bottom: 5vh;
+  width: 100vw;
+  margin: 0;
+  height: auto;
 }
 
 .footer-text {
-    height: 30px;
-    font-family: 'Space Mono';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 40px;
-    line-height: 30px;
-    color: #ffffff;
-    padding-left: 15%;
-    margin: 5%;
+  height: 30px;
+  font-family: 'Space Mono';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 30px;
+  color: #ffffff;
+  padding-left: 15%;
+  margin: 5%;
 }
 
 .hor-footer-links {
-    display: flex;
-    flex-direction: column;
-    margin-top: auto;
-}
-
-.hor-icon {
-    height: fit-content;
-    margin-bottom: 40px;
-}
-
-.hor-icon {
-    height: fit-content;
+  display: flex;
+  flex-direction: column;
+  margin-top: auto;
 }
 
 .sherm {
-    padding: 5%;
-    margin-top: 0;
+  padding: 5%;
+  margin-top: 0;
 }
 
 .logo-placement {
-    width: 14vh;
-    height: 18vh;
+  width: 14vh;
+  height: 18vh;
 }
 
 .hor-social-icons {
-    margin-top: auto;
-    display: flex;
-    flex-direction: column-reverse;
-}
-
-.footer-two {
-    background-color: orange;
-    align-items: right;
+  margin-top: auto;
+  display: flex;
+  flex-direction: column-reverse;
 }
 
 .rectangle {
-    display: inline-block;
-    width: 100px;
-    height: 100px;
-    background: black;
+  display: inline-block;
+  width: 100px;
+  height: 100px;
+  background: black;
 }
 
 .rectangleLogo {
-    margin-left: 50px;
-    margin-top: 20px;
-    width: 90px;
-    height: 90px;
-    background: black;
+  margin-left: 50px;
+  margin-top: 20px;
+  width: 90px;
+  height: 90px;
+  background: black;
+}
+
+.footer-container-mobile {
+  background-color: #187dff;
+  text-align: left;
+  padding-left: 15vw;
+  padding-top: 5vh;
+  padding-right: 15vw;
+  padding-bottom: 5vh;
+  width: 100vw;
+  margin: 0;
+  height: auto;
 }
 
 .social-media {
-    width: 2vw;
-    height: 3vh;
+  width: 1.5vw;
+  height: 5vh;
 }
 
 @media (max-width: 1200px) {
-    .social-media {
-        width: 4vw;
-        height: 9vh;
-    }
+  .social-media {
+    width: 3vw;
+    height: 5vh;
+  }
 }
 
 @media (max-width: 650px) {
-    .social-media {
-        width: 5vw;
-        height: 3.5vh;
-    }
+  .social-media {
+    width: 4vw;
+    height: 6vh;
+  }
+}
+
+@media (max-width: 650px) and (max-height: 400px) {
+  .social-media {
+    width: 4vw;
+    height: 11vh;
+  }
 }
 
 @media (max-device-width: 650px) {
-    .social-media {
-        width: 6vw;
-        height: 4vh;
-    }
-}
\ No newline at end of file
+  .social-media {
+    width: 6vw;
```

Comment:
this may possibly overwrite the existing max-width rule

Reply:
yeah this is on purpose they should be a bit bigger on mobile to be easier to tap on

Reply:
I just looked at it to make sure max-width doesnt work on mobile so max-width-device is needed. Though I am going to change it to match because it does look better at 6vw for both.